src/components/SignupFormContainer.jsx

```diff
@@ -99,7 +99,7 @@ export class SignupFormContainer extends Component {
           <InputField
             placeholder="Email"
             type="email"
-            id="email"
+            id="email2"
```

Comment:
I don't know why you changed this back to `email2`. This creates two problems:
1. There is a component that already has the id `email2` and ids are meant to be unique.
2. The email field on the Signup form would no longer work.

Kindly change this back.
cc @primuse @EfeAgare @omenkish

Reply:
Ok, Yes I thought I changed it back, but will do that now